x-pack/plugins/apm/public/components/alerting/register_apm_alerts.ts

```diff
@@ -23,6 +23,17 @@ export function registerApmAlerts(
       errors: [],
     }),
     requiresAppContext: true,
+    defaultActionMessage: i18n.translate(
+      'xpack.apm.alertTypes.errorCount.defaultActionMessage',
+      {
+        defaultMessage: `\\{\\{alertName\\}\\} alert is firing for the following configuration:
+
+- Service name: \\{\\{context.serviceName\\}\\}
+- Environment: \\{\\{context.environment\\}\\}
+- Triggered value: \\{\\{context.triggerValue\\}\\}
```

Comment:
We should have values
```suggestion
- Triggered value: \\{\\{context.triggerValue\\}\\} errors
```

Perhaps even with the interval:
```suggestion
- Triggered value: \\{\\{context.triggerValue\\}\\} errors over {{interval}} minutes
```